adaptors/library/jobs/satusehat-create-observations.js

```diff
@@ -0,0 +1,140 @@
+// Create an observation in Satusehat
+
+fn(state => {
+  function createObservationData(item, state, result) {
```

Comment:
I don't think the two functions here should be declared inside the `fn` block. Looks like a strange pattern to me